classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, :students
```

Comment:
- [ ] Following best practices, we only intend to _access (get)_ `students` object and therefore it should be marked with `attr_reader`. Marking it with `attr_accessor` may expose our object to unwanted modifications. Kindly ensure to improve upon this aspect.